src/style.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+}
+
+html {
+  height: 100%;
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #0f172a;
+}
+
+.container {
+  margin: 200px auto;
+  width: 500px;
+  height: auto;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+  border: 1px solid gainsboro;
+  border-radius: 10px;
+}
+
+.header {
+  display: flex;
```

Comment:
I  noticed you have several classes with the display **flex** attribute, you can create a class with display flex and add it to respective elements or tags

Reply:
hello @toyman640 
the idea of using display flex on most of my elements is to help me align my elements easily 
l probably think that adding a single selector with the display property set to flex and just adding it on my elements would make the work untidy don't you think so